src/stories/containers/finances-overview/useFinancesOverview.ts

```diff
@@ -1,10 +1,22 @@
 import { useThemeContext } from '@ses/core/context/ThemeContext';
+import { DateTime } from 'luxon';
+import { useState } from 'react';
 
 const useFinancesOverview = () => {
+  const actualYear = DateTime.local().minus({ year: 1 }).year;
+  const [year, setYear] = useState<number>(actualYear);
   const { isLight } = useThemeContext();
+  const years = [2021, 2022, 2023];
 
+  const handleChangeSelectYear = (year: number) => {
+    setYear(year);
+  };
   return {
     isLight,
+    year,
```

Comment:
could you use a more descriptive name here? (eg: selectedYear)

Reply:
done